pkg/eks/api.go

```diff
@@ -44,11 +44,16 @@ type ProviderServices struct {
 	eks  eksiface.EKSAPI
 	ec2  ec2iface.EC2API
 	sts  stsiface.STSAPI
+
+	cfnRoleArn string
```

Comment:
`s/Arn/ARN/`

Reply:
Turned out this was also redundant. Removed in 1fcf70d